libs/ledger-live-common/src/families/cosmos/api/types.ts

```diff
@@ -266,12 +266,15 @@ export type GetRedelegations = {
       validator_dst_address: string;
     };
     entries: {
-      creation_height: string;
-      completion_time: string;
-      initial_balance: string;
-      shares_dst: string;
-      unbonding_id: string;
-      unbonding_on_hold_ref_count: string;
+      redelegation_entry: {
+        creation_height: string;
```

Comment:
I think your mock does not response with a value for all those fields.
What is correct?

Reply:
Indeed nice catch, I copy pasted from the original PR from kevin

Reply:
Done